wiki(meandeck): add alternative piles chapter (#130)
angrybacon/doomsday-wiki: markdown/chapters/meandeck/alternative-piles.md

---
authors: Doishy
title: Alternative Pile Building
---

> ## Contents

Reviewer: ```suggestion
## Table of Contents
```

The `remark` plugin looks for `Table of Content` specifically 🤓

## Preamble

In this chapter we will present some alternative piles which may be utilised
depending on the tools available in your deck. This could include spicy main
deck choices or the use of certain sideboard cards and is here to make you aware
of how some of these tools are utilised in the wild. These are not listed in
any specific order.

## Shelldock Isle Piles

You might see a lot of people running card:[SI] alongside card:[Em] in their 
sideboards. The purpose of these cards is to use them in a pile that is 
effective against blue based control decks which do not run cards like
Wasteland. This pile often requires passing multiple turns and can be used
in conjunction with Oracle. 

> PTT

:::row{variant=pile}
- SI
- Em
- Island
- CoS
- TO
:::

In theory you can replace Emrakul with any card (including lands) however
the uncounterable nature, the take an extra turn on casting (which still 

> works with Shelldock!) and the attack for 15 with Annihilator 6 all make it

Reviewer: ```suggestion
works with Shelldock!) and the attack for 15 with *Annihilator 6* all make it
```

probably the most effective tool to use. 

> ## Piles using Dragon's Rage Channeler

Reviewer: ```suggestion
## Piles Using Dragon's Rage Channeler
```

Some people have tried various creature shells alongside the Doomsday package
and one of the more interesting ones for use as part of the Doomsday

> resolution is card:[Dragon's Rage Channeler] (DRC). Each spell cast can let you "dig"

Reviewer: ```suggestion
resolution is card:[Dragon's Rage Channeler] (DRC). Each spell cast can let you *dig*
```

Also, feel free to add an entry here `src/tools/scryfall/cards.ts` as the card makes sense both for and against Doomsday as an archetype (and it's a pain to type, I often forget where the `'s` actually is)

> one additional card deeper through your pile thanks to the Survail ability.

Reviewer: ```suggestion
one additional card deeper through your pile thanks to the *Surveil* ability.
```

Let's look at a couple of examples.

> DRC in play + non-Consider castable cantrip in hand

:::row{variant=pile}
- DA
- LED
- SW
- LP
- TO
:::

The most basic example above allows you to turn any regular cantrip into a 

> pseudo copy of card:[Con] by giving it a Surveil 1 effect attached. This allows

Reviewer: ```suggestion
pseudo copy of card:[Con] by giving it a *Surveil 1* effect attached. This allows
```

you to perform the basic Consider pile using either card:[Pnd], card:[Pre] or
card:[BS].

> {U} + DRC in play + non-Brainstorm castable cantrip in hand + X

:::row{variant=pile}
- SW
- BS
- LED
- EoA
- TO
:::

Here you can execute a Brainstorm pile even if you are at a low life total without the
use of card:[SW] or if you don't have access to tools like card:[IU] or card:[Con] which

> still allows you to get to 0 cards left in the deck. You can follow the listed steps:

Reviewer: ```suggestion
still allows you to get to 0 card left in the deck. You can follow the listed steps:
```

Author: Not sure why this change suggested, should be "cards"

1. Cast your cantrip and Surveil SW into the graveyard, draw BS
1. Cast BS, do not Surveil and put back TO and X
1. Cast LED, Surveil X into the graveyard
1. Cycle Edge, holding priority and cracking LED for {U}{U}{U} draw Oracle
1. Cast Thassa's Oracle (TO) with an empty library

## Double Oracle Piles

Often you will see people running 2 card:[TO] in the maindeck or 1 main and 1 in the side.
This card naturally fits into many pre-existing piles which we have covered already and give

> the option to "try and try again" when looking to beat cards like card:[Endurance].

Reviewer: ```suggestion
the option to *try and try again* when looking to beat cards like card:[Endurance] and :card[Memory's Journey].
```

> PTT + {U}{U}

:::row{variant=pile}
- IU
- LP
- TO
- CoS
- TO
:::

This is a very basic take on the standard card:[IU] pile which is often used. It can be 

> effective in "risking" jamming a pile without putting too many cards into the graveyard

Reviewer: ```suggestion
effective in *risking* jamming a pile without putting too many cards into the graveyard
```

which may be returned with Endurance and also still being able to have another Oracle
in reserve if the first one doesn't work out. What is nice is that even if they were to

> Endurance you here you library would become (randomised after the first TO) some variation

Reviewer: ```suggestion
Endurance you here, your library would become (randomised past the first TO, still on top of the library) some variation
```

of the following:

:::row{variant=pile}
- TO
- DD
- LP
- IU
:::

> As a result if they do cast and Endurance then you can pass and simply draw the second Oracle

Reviewer: ```suggestion
As a result if they do cast an Endurance then you can pass and simply draw the second Oracle
```

now and try to win with a Devotion of 4 with 3 cards remaining in your deck.
Other options include this example against a {U}{R} Delver deck which may run card:[Ragavan, Nimble Pilferer]
in. 

> PTT + Cantrip + {U}

:::row{variant=pile}
- SW
- CoS
- TO
- EoA
- TO
:::

Here you might have a scenario where you want to try and win quickly but suspect they may have a Red
Blast effect. In which case you can pass, allow Ragavan to exile SW, and try to jam the first
Oracle off of Cavern (assuming the cantrip resolves of course). If that Oracle is dealt with then
you can pass again, Ragavan exiles the card:[EoA] and you try casting the second Oracle.

> ## Slow Piles using Interactive Elements

Reviewer: ```suggestion
## Slow Piles Using Interactive Elements
```

These piles have been given a generic title but they generally refer to piles that will use tools
like card:[Baleful Strix], card:[Uro, Titan of Nature's Wrath] and card:[Teferi, Time Raveler] to 
act as speedbumps by blocking, gaining life or bouncing opposing permanents whilst also drawing
through the deck.

These typically pass multiple turns and rely on having a solid manabase already developed in play
so tend to be part of the slower Doomsday variants.

> PTT + {U}{G}{1}

:::row{variant=pile}
- Uro, Titan of Nature's Wrath
- Island
- Uro, Titan of Nature's Wrath
- CoS
- TO
:::

or

> PTT + {U}{B}

:::row{variant=pile}
- Baleful Strix
- EoA
- Baleful Strix
- CoS
- TO
:::

> You can also combine elements to use Teferi to bounce Strix for additional card draw.

Reviewer: ```suggestion
You can also combine elements to use Teferi to bounce Strix for additional card draw and still maintain a higher {U} Devotion.
```

> PTT + {U}{B}

:::row{variant=pile}
- Baleful Strix
- Tundra
- Teferi, Time Raveler
- CoS
- TO
:::

## Echo of Eons piles

You can use card:[EoE] as a larger version of Ideas Unbound/Deep Analysis which also interacts 
favourably with Consider. This is not a commonly used card but if you wanted to maybe try
and utilise storm cards or perhaps run additional LEDs then it might be useful.
In order for these piles to work and not kill you, you must ensure you have at least one
additional card available either in hand or that you can put into your graveyard (like
a card:[LP], Fetchland or land you can sacrifice with Edge of Autumn in addition to the 
original Doomsday cast which will be in your graveyard and the 5 cards comprising your deck.
As the Echo will exile itself you cannot count that as one of your cards.

> PTT + Echo of Eons in Hand + Cantrip

:::row{variant=pile}
- LED
- PoN
- TO
- LP
- LP
:::

> There is a much more extensive look at Echo of Eons in the DDFT section of the wiki.

Reviewer: ```suggestion
There is a much more extensive look at Echo of Eons in the [DDFT section][ddft:echo] of the wiki.

[ddft:echo]: /ddft/echo
```

## Next Steps

You don't need to have anything here memorised but do use the information presented

> here as a way of looking at alternative ways perhaps you can build piles with either

Reviewer: ```suggestion
here as a way of looking at alternative ways. Perhaps you can build piles with either
```
